frontend/src/components/Forms/OtherForm/index.js

```diff
@@ -0,0 +1,26 @@
+import { PropTypes } from "prop-types";
```

Comment:
I don't think you need this component at all.

You can achieve the same adding `justify-content: center;` to your `otherInterestComponent` class in `SignupPage`